webapp/src/ts/modules/contacts/contacts.component.ts

```diff
@@ -310,6 +310,52 @@ export class ContactsComponent implements OnInit, AfterViewInit, OnDestroy {
     return this.sortDirection === 'last_visited_date';
   }
 
+  private getContactsDocIds(contactsList, usersHomePlace): string[] {
+    const docIds = contactsList.map((item) => item._id);
```

Comment:
This one is missing the `if (options.withIds) {` that it used to have originally. 
Is that intentional?  If not, it would be good to have an unit test catching that